[Coding Guidelines] Add notetaker role

PLeVasseur opened the pull request:
Adding:
* Meeting policies we agreed to at the London Consortium meeting
* A notetaker role to formalize a bit what we were already doing in the Coding Guidelines Subcomittee

JoelMarcey reviewed (commented):
I wonder if we should up-level these to the parent folder as consortium-wide? I guess we can duplicate the information across all subcommittees, but should we?

PLeVasseur commented:
I tried to up level the bits we agreed to consortium-wide into the Meeting Policies. I suppose I was trying to not speak for how others are or will run subcommittees. However, given I am suggesting this role I am fine with commonizing.

What do you think @alexandruradovici ?

JoelMarcey commented:
Yeah, the README.md stuff looks good. I was thinking about the note taking. Maybe it's ok if subcommittees have different note taking policies, but I think consistency would be nice.

alexandruradovici commented on subcommittee/coding-guidelines/notetaker-role.md:
```diff
@@ -0,0 +1,53 @@
+
+## Take meeting notes live in a Google Docs document
+
+During the meeting you'll take meeting minutes of participants' discussion in a Google Docs document that will be shared as the meeting begins.
```
What about adding a link to an empty read-only template that meeting notes takers can copy?

PLeVasseur replied:
I like that idea! Hows [this](https://docs.google.com/document/d/1vx_Ih9DRXQrHU_pP_uUZWjqvMKzzAR6EkS5lINzT4bA/edit?usp=sharing) look?

If it looks good I can incorporate this into `notetaker-role.md`, add this step, and then move the file up a level to the consortium-level.

cc @alexandruradovici @JoelMarcey

JoelMarcey replied:
Yep. Looks good!

alexandruradovici commented:
I suggest making this document consortium-wide, not only for the coding guidelines.

alexandruradovici commented on subcommittee/coding-guidelines/notetaker-role.md:
```diff
@@ -0,0 +1,53 @@
+
+## Submit the meeting minutes to the repo
+
+Following the meeting you will select all the contents of the Google Doc, right click, and then click `Copy as Markdown` on the menu that pops up.
```
```suggestion
Following the meeting you will select all the contents of the Google Doc, right click, and then click `Copy as Markdown` on the menu that pops up. You have to make sure Markdown is enabled for the document. Enabling it can be done by checking **Tools -> Preferences -> Enable markdown**.
```

PLeVasseur commented:
I refreshed this based on feedback.
* Moved notetaker role up to consortium-wide
* Added link to starter notes that can be copied

cc @alexandruradovici @JoelMarcey

JoelMarcey commented:
Couldn't you just have a directory called `docs` instead of `consortium-wide/docs`? Isn't it implied that anything in a `docs` directory would apply to the entire consortium as a whole?

PLeVasseur commented:
> Couldn't you just have a directory called docs instead of consortium-wide/docs? Isn't it implied that anything in a docs directory would apply to the entire consortium as a whole?

Done!
